docker/appserver/Tomcat/example/pom.xml

```diff
@@ -63,27 +63,39 @@
 						</configuration>
 					</execution>
 					<execution>
-						<id>copy-configuration</id>
+						<id>unpack-resources</id>
```

Comment:
Ik weet niet of ik het zo'n goed idee vind om configuratie en properties uit te pakken in plaats van te kopieren als .jar files

Reply:
Aangepast dankzij classloader